lib/ruby-stackoverflow/client/parse_options.rb

```diff
@@ -1,24 +1,22 @@
 module RubyStackoverflow
-  class Client
-    module ParseOptions
-      def parse_options(options = {})
-        options.each do|k,v|
-          case k
-          when :fromdate , :todate, :min, :max
-            begin
-              options[k] = Time.parse(v).to_i
-            rescue
-              options[k] = v
-            end
-          else
+  module ParseOptions
+    def parse_options(options = {})
+      options.each do|k,v|
+        case k
+        when :fromdate , :todate, :min, :max
+          begin
+            options[k] = Time.parse(v).to_i
+          rescue
             options[k] = v
           end
+        else
+          options[k] = v
         end
       end
+    end
 
-      def join_ids(ids)
-        ids.join(';')
-      end
+    def join_ids(ids)
+      ids.join(';')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.